.github/workflows/COMMENT_DEVELOP_FIX.yml

```diff
@@ -11,6 +11,8 @@ jobs:
   comment:
     name: Comment on fix to develop
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
Could you share where this "write" is documented? I thought until today that this is about commit (push) access.

Reply:
I found [this](https://github.com/orgs/community/discussions/26644), but it is not official documentation.

Reply:
Sure, it's documented at https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-pull-requests